docs.css

```diff
@@ -429,6 +429,18 @@ footer{
     border: 1px solid #3B82F6;
 }
 
+.iconBtn{
```

Comment:
It would be great if you add hover shadow to it

Reply:
Done ! Thanks